src/validate.rs

```diff
@@ -140,7 +141,7 @@ fn test_invalid_stop_points() {
             object_name: None,
             related_objects: vec![],
             details: Some(
-                "error while reading stops.txt: CSV deserialize error: record 12 (line: 13, byte: 739): invalid float literal".to_string()
+                "error while reading stops.txt: CSV deserialize error: record 12 (line: 13, byte: 739): invalid float literal: invalid float literal".to_string()
```

Comment:
invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literalinvalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal invalid float literal: invalid float literal  invalid float literal: invalid float literal  invalid float literal: invalid float literal

Reply:
gnagnagna

The whole point of those updates is to give better error messages, but this will be handled in another PR (with a specific error handling when given a position)

I was thinking about your limited attention period but if you want I can make it in the same PR :stuck_out_tongue_winking_eye: